Android build failure (Could not determine java version)

`gradelw` calls fail with messages:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine java version from '11.0.9'.
```

Need to upgrade used gradle version to support JDK 11.
- 5.0+ according to https://docs.gradle.org/5.0/release-notes.html#java-11-runtime-support

--- a/platforms/android/gradle-wrapper/gradle/wrapper/gradle-wrapper.properties
+++ b/platforms/android/gradle-wrapper/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

